cosmovisor/go.mod

```diff
@@ -108,7 +101,7 @@ require (
 	github.com/tendermint/btcd v0.1.1 // indirect
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15 // indirect
 	github.com/tendermint/go-amino v0.16.0 // indirect
-	github.com/tendermint/tendermint v0.34.20 // indirect
+	github.com/tendermint/tendermint v0.35.9 // indirect
```

Comment:
we should figure out where this upgrade is coming from. We should have the whole repo on 0.34.20

Reply:
Yes, I was checking exactly that in https://github.com/cosmos/cosmos-sdk/pull/12741. It turns out if we `go mod tidy` in `./cosmovisor` right now we get an error. If we remove these indirect imports and `go mod tidy` it will put it back to `0.34.20`. However, as soon as we `go work sync` this forces cosmovisor to have tm `0.35.9` as indirect dependency.

Reply:
I don't think we need cosmovisor to be included in the `go.work` anyway, as `cosmovisor` is not used by the sdk. Investigating...